mucommander-translator/src/main/resources/dictionary.properties

```diff
@@ -484,6 +484,12 @@ search_dialog.case_sensitive = $[file_selection_dialog.case_sensitive]
 search_dialog.matches_regexp = $[file_selection_dialog.matches_regexp]
 search_dialog.wildcards = (* = any string, ? = any character, \\ = escape for literals: *?)
 search_dialog.search_text  = Containing text
+search_dialog.size=Size
```

Comment:
you can rather refer to existing translation here and in the units below with, e.g., $[size]